public/styles.css

```diff
@@ -18,69 +18,80 @@ h3,
 h4,
 h5,
 h6 {
-  font-weight: bold !important;
+  font-family: "M PLUS Rounded 1c", Helvetica, sans-serif;
+}
+
+p,
+li,
+span {
+  font-family: "Figtree", Arial, sans-serif;
 }
 
 h1 {
   font-size: 64px !important;
-  @media (max-width: 576px) {
-    font-size: 54px !important;
-  }
 }
 
 h3 {
   font-size: 48px !important;
-  @media (max-width: 576px) {
-    font-size: 40px !important;
-  }
 }
 h4 {
   font-size: 40px !important;
-  @media (max-width: 576px) {
-    font-size: 30px !important;
-  }
 }
 
 h5 {
   font-size: 32px !important;
-  @media (max-width: 576px) {
-    font-size: 26px !important;
-  }
 }
 
 h6 {
   font-size: 24px !important;
-  @media (max-width: 576px) {
-    font-size: 18px !important;
-  }
 }
 
-p {
-  font-size: 16px;
-  @media (max-width: 576px) {
-    font-size: 14px;
-  }
+.section-title {
+  color: var(--concrete);
 }
 
-/* CARDS */
-
 .card {
   border-width: 10px !important;
   border-radius: 20px !important;
 }
 
-.square-card {
-  .card-img-top {
-    border-bottom: none;
-    object-fit: cover;
-    border-radius: 20px;
-  }
+.card-img-top {
+  border-bottom: none;
+  aspect-ratio: 1 / 1;
+  object-fit: cover;
+  border: 20px solid white !important;
+  border-bottom: none !important;
+  --bs-card-inner-border-radius: calc(
+    20px - (var(--bs-border-width))
+  ) !important;
+  width: 100%;
+}
 
-  .card-title {
-    font-size: 16px;
-    font-weight: bold;
-    color: var(--concrete);
-  }
+/*NAVBAR*/
+
+#navbar {
+  background-color: #6982c3;
```

Comment:
We should use the color variables (forgot to change this one). So, for this instead of using the hex code we would use ``var(--persion-jewel)``